cdap-api/src/main/java/co/cask/cdap/api/workflow/WorkflowForkNode.java

```diff
@@ -19,13 +19,13 @@
 import java.util.List;
 
 /**
- * Represents the FORK node in the {@link Workflow}
+ * Represents the FORK node in the {@link Workflow}.
  */
 public class WorkflowForkNode extends WorkflowNode {
   private final List<List<WorkflowNode>> branches;
 
   public WorkflowForkNode(String nodeId, List<List<WorkflowNode>> branches) {
-    super(nodeId, WorkflowNodeType.FORK);
+    super(nodeId, WorkflowNodeType.FORK, null);
```

Comment:
`fork` is artificial node in the Workflow, which does not do any processing. So the parent id is not added to the fork. Parent ids of the first node in the fork branches contains the previous node of the fork.